src/runtime/less/ts-toolbar.less

```diff
@@ -94,6 +94,7 @@
 	padding-left: 0;
 	padding-right: 0;
 	text-overflow: ellipsis;
+	max-width: 8 * @ts-unit-triple;
```

Comment:
How would this work if the screen is smaller than `8 * @ts-unit-triple` (for example on a mobile phone). Would the title then overlap the buttons all over again?